messages/commands/water-jet_steering.json

```
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Water-jet steering",
+    "description": "Command given for water-jet steering unit",
+    "required": [
+        "command_id",
+        "bucket_angle"
+    ],
+    "properties": {
+        "command_id": {
+            "type": "string",
+            "title": "Command id",
+            "description": "Platform unique command id"
+        },
+        "angle": {
+            "type": "number",
+            "title": "Water-jet angle set",
+            "description": "Water-jet angle in degrees, positive when pushing water to starboard"
```

```suggestion
            "description": "Water-jet angle in degrees, positive when pushing water to port"
```

To conform to the same standard as the rudder.

---

Are port turns positive? Really? Not following the compass? This has to be some naval architect thing 😂